groups/cdn/cloudfront.tf

```diff
@@ -53,6 +53,31 @@ resource "aws_cloudfront_origin_access_control" "assets" {
   signing_behavior                  = "always"
   signing_protocol                  = "sigv4"
 }
+resource "aws_s3_bucket" "log_bucket" {
+  bucket = "cloudfront-logs-bucket"
+
+  block_public_acls   = true
+  block_public_policy = true
+
+  lifecycle_rule {
+    id      = "log-retention"
+    enabled = true
+
+    expiration {
+      days = 30
+    }
+  }
+
+  tags = {
+    Name = "Cloudfront Logs Bucket"
+  }
+}
+
+logging_config {
+  bucket          = aws_s3_bucket.log_bucket.bucket_domain_name 
```

Comment:
I do not think .bucket_domain_name will work here. I would have thought you need to use .s3_bucket_id